src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -28,7 +28,7 @@
     <OutputType>Library</OutputType>
   </PropertyGroup>
   <PropertyGroup Condition="'$(IsCoreClr)' == 'false'">
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
>v4.6.1 [](start = 28, length = 6)

Are you missing an `App.config` change to make sure users get a nice error message if they have an older CLR installed?

Reply:
And make sure the change makes it to your OpenDebugAD7.exe.config - it should, but good to verify.

---
In reply to: [217172144](https://github.com/Microsoft/MIEngine/pull/757#discussion_r217172144) [](ancestors = 217172144)